README.md

```diff
@@ -4,6 +4,20 @@ The most bestestest tools for make running the census big success wow.
 
 ## How to use - Message Manipulator
 
+The message manipulator will allow you to look at rabbit messages inside a queue you specify.
+### Arguments
+
+| Name                      | Description                                                                                                         |                                                                                        
+| ---------------------     | ------------------------------------------------------------------------------------------------------------------- |
+| `source_queue_name`       | Rabbit queue to find messages on                                                                                    |
+| `message_hash_search`     | Rabbit message hash to search by                                                                                    |
+| `-s --search`             | Search for a string inside of the rabbit message body                                                               |                                                  
```

Comment:
Is it worth mentioning the `-l --limit` flags here as well? I guess the default value is fine for most situations though...

Reply:
I've left it out because in Adams recent PR that was merged, it was removed and hardcoded to 100

Reply:
Ah sorry, I don't think I've pulled from this repo in a while!